src/graphql/types.ts

```diff
@@ -176,6 +176,12 @@ export enum ColonyType {
 }
 
 
+export type ModelUserConnection = {
```

Comment:
I have no idea why this file changed. As watchlist query does indeed exist. Must be some amplify codegen "unpleasantness"